applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -433,9 +435,40 @@ def _GetAdjointParameters(self):
             with open(self.response_settings["adjoint_settings"].GetString(),'r') as parameter_file:
                 adjoint_parameters = Parameters( parameter_file.read() )
 
+            # prevent the case that primal solution is transfered by python and the HDF5 process
+            if self.use_python_primal_solution_data_transfer:
+                if adjoint_parameters["processes"].Has("list_other_processes"):
```

Comment:
This is a very weak mechanism since the names can be random!

Reply:
I think we could even remove this check, as nothing happens, just the variables are transferred twice. Especially if the new setting is mandatory i think we can avoid this (by the way also the existing check above in line 375

Reply:
Then the only necessary check is that "auto" can only be used if data is transferred via python